src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk

```diff
@@ -27,9 +27,15 @@
 
 {% macro paidInFullClaimantClaimSettledDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ defendantName }} paid you {{ totalAmountTillToday }} on {{ moneyReceivedOn }}.',
-      { defendantName: claim.claimData.defendant.name, totalAmountTillToday: claim.totalAmountTillToday | numeral,
-        moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+    {% if claim.response.amount %}
+        <p>{{ t('{{ defendantName }} paid you {{ amount }} on {{ moneyReceivedOn }}.',
+            { defendantName: claim.claimData.defendant.name, amount: claim.response.amount | numeral,
```

Comment:
Can key/value pair 
```
defendantName: claim.claimData.defendant.name,
moneyReceivedOn: claim.moneyReceivedOn | date
```
be extracted out of if/else block instead repeating them?